api/spec/v1.yaml

```diff
@@ -1357,6 +1357,12 @@ components:
           format: int32
           description: Number of transactions in the block.
           example: 17
+        proposer:
```

Comment:
I would expect to see these beauts in e2e diffs. Probably just needs to be re-run?

Reply:
Ah yes, I see e2e fails with exactly the expected diffs :+1: